sql/hive/pom.xml

```diff
@@ -36,6 +36,12 @@
   </properties>
 
   <dependencies>
+    <!-- Added for sbt issue, expected to be evicted -->
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>2.10.3</version>
```

Comment:
My issue was different (mismatched junit versions) and the fix did required a pom change (to update the version of junit). I haven't really looked into this particular one, but hive does not seem to depend directly on scala anywhere, so I'm not sure where the 2.10.3 dependency might be coming from.